### PR TITLE
Update how-to-build-and-run.md

### PR DESCRIPTION
**Pull request by Tbkhi**
Adds examples of `./x.py` commands.

### CODE REVIEW

File: src/building/how-to-build-and-run.md [hunk 1]
```diff
@@ -120,6 +120,16 @@ You can avoid this error by allowing powershell to run local scripts:
...
+## Other `x` commands
+
+Here are serveral non-comprehensive examples and explainations for various
+`x.py` commands:
+
+- `./x.py clean` - invalidates the build config cache (but not LLVM cache also)
+- `./x.py check --all-targets` - does cross-compilation for a lot of different targets
+- `./x.py test --stage 1 --keep-stage-std 1` - use a cached version of the compiler
+- `./x.py test --bless src/test/mir_opt - automatically generates `MIR` dumps
+- `./x.py test --stage 1 --compare-mode chalk` - sets compare-mode to `chalk`
```
Comment (tshepang) [inline src/building/how-to-build-and-run.md:132]: we decided to standardize on `./x` in the docs (see #1707)

Comment (Tbkhi) [reply src/building/how-to-build-and-run.md:132]: Closing PR as obsolete.

**Comment by Tbkhi** [comment]
Obsolete.

File: src/building/how-to-build-and-run.md (hunk 1 above)
Comment (tshepang) [reply src/building/how-to-build-and-run.md:132]: why is it obsolete @Tbkhi

**Pull request closed without merging**